samples/sampleAPIcall.js

```diff
@@ -0,0 +1,8 @@
+import { getGameSession } from "./queries";
+import { API, graphqlOperation } from 'aws-amplify';
+
+export const getGameSession = async () => {
```

Comment:
wouldn't this need an id?

Reply:
Yes this is my bad, this should have been listGameSessions, but I'll push both to avoid any confusion